assets/css/style.scss

```diff
@@ -247,18 +247,23 @@ Divs
 *******************************************************************************/
 
 .resource-row {
-  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
 }
-
 .resource-div {
-  width: 240px;
-  padding: 20px;
-  margin: 10px;
-  background-color: white;
-  display: inline-block;
-  text-align: center;
-  vertical-align: top;
-  box-shadow: 0px 0px 10px #888888;
+ display: flex;
+ flex-direction: column;
+ width: 44%;
+ padding: 13px;
+ margin: 4px 4px 4px 4px;
+ background-color: white;
+ text-align: center;
+ box-shadow: 0px 0px 10px #888

```

Comment:
Nit: hex value got changed and missing semicolon

Reply:
reverted